libraries/AP_TemperatureSensor/AP_TemperatureSensor_Params.cpp

```diff
@@ -58,7 +58,7 @@ const AP_Param::GroupInfo AP_TemperatureSensor_Params::var_info[] = {
     // @Param: SRC
     // @DisplayName: Sensor Source
     // @Description: Sensor Source is used to designate which device's temperature report will be replaced by this temperature sensor's data. If 0 (None) then the data is only available via log. In the future a new Motor temperature report will be created for returning data directly.
-    // @Values: 0: None, 1:ESC, 2:Motor(not implemented yet), 3:Battery Index, 4:Battery ID/SerialNumber, 5: Pitot tube
+    // @Values: 0: None, 1:ESC, 2:Motor(not implemented yet), 3:Battery Index, 4:Battery ID/SerialNumber, 5: CAN based Pitot tube
```

Comment:
@magicrub says the pitot tube option isn't implemented yet, so lets just remove it from the list

Reply:
yup, lets remove "Pitot tube" enum for now until I get my AP_Heater library merged in which will use it. The enum is currently a placeholder but I agree that it's creating confusion right now.

Reply:
@tridge but this is only part of a larger code change...and the enum impacts this code, not just the switch statement.....its the reason I said this is CAN specific: https://github.com/ArduPilot/ardupilot/pull/24462
should that CAN change be reverted also?

Reply:
I posted a PR that removes the unused ENUM as Tridge suggested...you guys decide